### PR TITLE
[CI:DOCS] Fix up podmansh man page

### PR DESCRIPTION
**Pull request by rhatdan**
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: docs/source/markdown/podmansh.1.md
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podmansh.1.md:30]: Old, but I think it's `Podman quadlet` elsewhere.

File: docs/source/markdown/podmansh.1.md
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podmansh.1.md:45]: Just double checkins, IDK, but is this change correct?

**Comment by TomSweeneyRedHat** [comment]
One question, otherwise LGTM

File: docs/source/markdown/podmansh.1.md
Comment (rhatdan) [reply docs/source/markdown/podmansh.1.md:45]: Yes this is the major reason for the change.  I am working on testing out the three scenarios and beginning to write a blog/howto for using podmanh.

**Comment by edsantiago** [comment]
/hold

Bugs found. Do not merge.

File: docs/source/markdown/podmansh.1.md
Comment (edsantiago) [inline docs/source/markdown/podmansh.1.md:37]: ...and the `sudo` here suggests you are not. Pick one or the other. I super-strongly suggest picking root, because `sudo cat > /path/only/writable/by/root` does not work, you need to do the horrible `| tee` thing.

Suggestion: remove the `sudo`, and add an explicit "run these commands as root" to line 33-ish.

File: docs/source/markdown/podmansh.1.md
Comment (edsantiago) [inline docs/source/markdown/podmansh.1.md:62]: This is evil.

This section (all the man page actually) is written in the "copy and paste this into your terminal" form, vs the "type these commands" form. I despise the copy-paste form, and this is why: the output of the grep is not clearly demarcated.

Suggestion: remove the `sudo`, and add explicit hash prompts to all commands, making it unambiguous that these are commands run as root. (And also making it obvious that the "confinedu" line is output from grep).

**Review by edsantiago — CHANGES_REQUESTED** [review]
Remove all sudos. Add `#` prefixes to commands. Add a note saying "run these as root".

**Review by lsm5 — APPROVED** [review]
LGTM after @edsantiago and @TomSweeneyRedHat  's comments addressed.

File: docs/source/markdown/podmansh.1.md
Comment (cevich) [inline docs/source/markdown/podmansh.1.md:96]: Very minor nit: Maybe I'm missing the reason, but my eye is drawn to the example user names.  Since they're not generic like `exmple1` or `someguy`, it makes me wonder if there's some name significance.  If there isn't, would you consider using more generic example names?

Comment (edsantiago) [reply docs/source/markdown/podmansh.1.md:96]: There _is_ significance. I like them, they make it immediately obvious what you're doing. I say keep them as they are.

**Review by edsantiago — APPROVED** [review]

File: docs/source/markdown/podmansh.1.md
Comment (edsantiago) [inline docs/source/markdown/podmansh.1.md:57]: Typo, missing 'n' in 'runnig'. And I think it would read better as:
>, and **ARE** able to read/write ...

**Review by edsantiago — CHANGES_REQUESTED** [review]
Sorry.

File: docs/source/markdown/podmansh.1.md
Comment (cevich) [reply docs/source/markdown/podmansh.1.md:57]: Maybe "Host's user account(s)"?

File: docs/source/markdown/podmansh.1.md
Comment (cevich) [reply docs/source/markdown/podmansh.1.md:96]: I did not look that closely TBH, so if Ed says they're fine, I'm comfortable with that.

File: docs/source/markdown/podmansh.1.md
Comment (rhatdan) [reply docs/source/markdown/podmansh.1.md:62]: Fixed

File: docs/source/markdown/podmansh.1.md
Comment (edsantiago) [inline docs/source/markdown/podmansh.1.md:59]: rather **than**, not then. And "home directory" is two words. I'm sorry! But I'll approve anyway, let you decide whether to fix now or in a followup.

**Pull request merged**
